week-2/Homework/mandatory/2-function-creation.js

```diff
@@ -37,7 +50,14 @@ Write a function that:
 - numbers greater 100 must be replaced with 100
 */
 
-function formatPercentage(arr) {}
+function formatPercentage(arr) {
+  for (let i = 0; i < arr.length; i++) {
+    arr[i] = number(arr[i].toFixed(2));
+    if (arr[i] > 100) arr[i] = 100;
+    arr[i] = arr[i] + "%";
```

Comment:
another way of coding arr[i] = arr[i] + "%"; is:
                                     arr[i] = arr[i].concat("%");